.github/workflows/java-continuous-integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '11' ]
+        java: [ '11', '17' ]
```

Comment:
Is it mandatory to keep java 11, since I think we are no longer using it ?

Reply:
We are only using java 17 for sonar-scanner.
Matrix defines tests for the cnes-report itself, which can run on java 11 (even if its now deprecated by SonarQube)